ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileClustersTable.tsx

```diff
@@ -129,18 +135,25 @@ function ProfileClustersTable({
                                             </Link>
                                         </Td>
                                         <Td dataLabel="Last scanned">{firstDiscoveredAsPhrase}</Td>
+                                        <Td dataLabel="Pass status">
+                                            <StatusCountIcon
+                                                text="check"
+                                                status="pass"
+                                                count={passCount}
+                                            />
+                                        </Td>
                                         <Td dataLabel="Fail status">
                                             <StatusCountIcon
                                                 text="check"
                                                 status="fail"
                                                 count={failCount}
                                             />
                                         </Td>
-                                        <Td dataLabel="Pass status">
+                                        <Td dataLabel="Manual status" modifier="fitContent">
```

Comment:
I think the "fitContent" here is making the column shrink to size which results in the header sometimes getting cutoff. I wonder if adding "fitContent" to everything (`Th`+`Td`) but the last compliance percentage makes sense?

![Screenshot 2024-06-25 at 10 37 24 AM](https://github.com/stackrox/stackrox/assets/61400697/a2b8e92e-ccb1-48b2-82ce-97847c319708)

Reply:
At the risk of not having a lot of context on this change, I noticed the comment about headers getting cut off and can say I use ` <Th modifier="nowrap">` more often then not to ensure the table column has a min-width equal to the length of the table header.

Reply:
Thank you for sharing your experience, David.

I suspect this “solution” might depend on happy coincidence that content in `th` and `td` elements have similar width.